src/configuration/app/multi-app.md

```diff
@@ -87,6 +87,10 @@ This will result (if we consider we are on the `http://example.com` domain):
 * `http://example.com/` and all the urls below it to be served by the AngularJS
 one.
 
+> **note**
+> In the specific case of a prefix, the development environments will be accessible by adding three `-` instead of a `.`:
```

Comment:
I mean URL prefix `https://foo.{default}` in comparison to URL suffix `https://{default}/foo/.